amazon-msk/cdktf/iot-ingest-and-control/zilla.yaml.mustache

```diff
@@ -0,0 +1,115 @@
+name: {{name}}
+vaults:
+  secure:
+    type: aws-secrets
+telemetry:
+  metrics:
+    - stream.active.received
+    - stream.active.sent
+    - stream.opens.received
+    - stream.opens.sent
+    - stream.data.received
+    - stream.data.sent
+    - stream.errors.received
+    - stream.errors.sent
+    - stream.closes.received
+    - stream.closes.sent
+  exporters:
+    stdout_logs_exporter:
+      type: stdout
+{{#cloudwatch}}
+    aws0:
+      type: aws-cloudwatch
+      options:
+        logs:
+          group: {{cloudwatch.logs.group}}
+          stream: events
+        metrics:
+          namespace: {{cloudwatch.metrics.namespace}}
+{{/cloudwatch}}
```

Comment:
This looks pretty clean now. 👍 

Is there a case where we might want `cloudwatch.metrics` but not `cloudwatch.logs` and vice versa?